validate_sample.py

```diff
@@ -30,9 +30,8 @@ class SampleValidator:
 
     def __init__(self):
         self.schema = {
-            'ARID': [mandatory(), max_length(21), unique()],
-            'ESTAB_ARID': [mandatory(), max_length(21)],
-            'UPRN': [mandatory(), max_length(12), numeric()],
+            'UPRN': [mandatory(), max_length(12), numeric(), unique()],
```

Comment:
Are we sure it's valid to make UPRN `unique`? I don't think this is correct, rows like CE units can have the same UPRN.